exercises/hamming/canonical-data.json

```diff
@@ -1,5 +1,5 @@
 {
-"exercise": "hamming",
+  "exercise": "hamming",
   "comments": [
     "Language implementations vary on the issue of unequal length strands.",
     "A language may elect to simplify this task by only presenting equal",
```

Comment:
The leading spacing here looks a bit odd. But I'm on mobile so not sure and it is not worth blocking this PR for.